crates/html-validation/src/lib.rs

```diff
@@ -17,6 +35,88 @@ lazy_static! {
     .iter()
     .cloned()
     .collect();
+
+    // list of svg elements
+    //  https://developer.mozilla.org/en-US/docs/Web/SVG/Element
+    //  a hashmap of `(tag, is_self_closing)`
+    static ref SVG_NAMESPACED_TAGS: HashMap<&'static str, bool> = [
+        ("a", true),
+        ("animate", true),
+        ("animateMotion", false),
+        ("animateTransform", true),   
+        ("circle", true),
+        ("clipPath",false),
+        //("color-profile",),
```

Comment:
For everything that is commented out in here mind leaving a comment explaining why the item is commented out?

Reply:
I should file an issue how are tags that has a `-` (dash) in between are done? How is it also done in attributes?